contracts/asset-proxy/contracts/src/MixinAuthorizable.sol

```diff
@@ -18,7 +18,7 @@
 
 pragma solidity ^0.5.9;
 
-import "@0x/contracts-utils/contracts/src/Ownable.sol";
+import "../archive/Ownable.sol";
```

Comment:
Hmmm.

I think this file should be archived too. New asset proxies will use `@0x/utils/Authorizable`, right?